daily_word_bot/db.py

```diff
@@ -11,30 +12,36 @@ def __init__(self, host: str):
         port: int = 6379
         self.r: redis.Redis = redis.Redis(host=host, port=port)
 
-    def save_user(self, message: Message):
+    def save_user(self, message: Message, user_levels: list = utils.POSSIBLE_USER_LEVELS):
```

Comment:
let's rather not assign default variables for list arguments., because ...
1. the potential flaw of having objects as default (commented on the previous review)
2. we move business logic to app, so classes have less cross module responsibility. Ideally DAO does not know anything about business logic just manages the db.